wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/convert/component/plugin/PluginComponentToWorkspaceApplier.java

```diff
@@ -72,6 +72,13 @@ public void apply(
     final String pluginId = pluginComponent.getId();
     final String registryUrl = pluginComponent.getRegistryUrl();
 
+    //    try {
+    //      URL tryUrl = new URL(pluginComponent.getId());
+    //      pluginComponent.setReference(pluginComponent.getId());
+    //    } catch (MalformedURLException e) {
+    //      // If we can't parse the ID as a URL it must not be a reference type plugin
+    //    }
+
```

Comment:
do we really need this commented block?

Reply:
Thanks for pointing that out, I forgot to remove this.